### DISCUSSION TITLE
Change url for panel

### DISCUSSION THREAD

**Discussion by aka-charos** [swizzin/swizzin#826] [Q&A + Help] [answered]
Hi,

By default panel url is set for root path (exampledomain.com/). How can we change the url to some path (eg. exampledomain.com/panel) ? 
Changing the location / in panel.conf (nginx) won't work .I assume tha is due to some proxy_set_header -but I am unaware what these lines control.

Any suggestions?

**Comment by liaralabs** [comment, marked as answer]
https://swizzin.ltd/applications/panel/#configuration-options

You need to set urlbase and change the proxy location as well
